service/frontend/workflowHandler.go

```diff
@@ -2256,6 +2260,19 @@ func (wh *WorkflowHandler) getHistory(domainID string, execution gen.WorkflowExe
 		return nil, nil, err
 	}
 
+	if response != nil {
+		wh.metricsClient.RecordTimer(scope, metrics.HistorySize, time.Duration(response.Size))
```

Comment:
Is it possible to also include domain here?

Reply:
Currently we can't emit domain as a tag.  Added a log instead.